include/functions.h

```diff
@@ -370,6 +370,8 @@ void create_part_shadow(s32 actorID, s32 partIndex);
 void remove_part_shadow(s32 actorID, s32 partIndex);
 void create_part_shadow_by_ref(UNK_TYPE arg0, ActorPart* part); // arg0 unused
 
+extern func_80266D6C(ActorPart*, UNK_TYPE arg0); //arg0 unused
```

Comment:
Don't `extern` functions in functions.h - It is throwing a bunch of errors since you didn't specify a return type for the function